dataframe-geo/src/test/kotlin/org/jetbrains/kotlinx/dataframe/geo/io/IOTest.kt

```diff
@@ -35,17 +36,18 @@ class IOTest {
         assert(geodf.crs == null)
     }
 
-    /*
-    TODO: doesn't work for now - writers adds ids that breaks order when read features
     @Test
     fun writeGeoJson() {
         val tempFile = Files.createTempFile("simple_points", ".json").toFile()
         simplePointsGeoDf.writeGeoJson(tempFile)
 
-        assertEquals(simplePointsGeoDf, GeoDataFrame.readGeoJson(tempFile.toURI().toURL()))
+        val loadedGeoDataFrame = GeoDataFrame.readGeoJson(tempFile.toURI().toURL())
+        assertEquals(simplePointsGeoDf.df, loadedGeoDataFrame.df)
+        // TODO: Doesn't work because of how equality between CRS is checked by geotools
```

Comment:
The proper way to check CRS equality in Geotools is 
`CRS.equals`, or `CRS.equalsIgnoreMetadata` if you don't want to compare metedata.

But there, shouldn't they be equal?

Reply:
Wait, then there's problem here! In GeoDataFrame (as in GeoJson specification) longitude should always go first! That's important!